src/test_board.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+from board import construct_board
+
+class BoardTest (unittest.TestCase):
+    def test_if_construct_board_return_correctly_sized_array(self):
+        board = construct_board()
+        self.assertEqual(len(board), 6)
+
+        for x, line in enumerate(board, 0):
```

Comment:
do not use loop statements in a test, it is not readable. Prefer to build your expected result and compare it with your function. If you do not know the Given/When/Then pattern, I advise you to have a look.